tests/end2end/features/navigate.feature

```diff
@@ -65,3 +65,8 @@ Feature: Using :navigate
         And I open data/numbers/1.txt
         And I run :navigate increment
         Then the error "No number found in URL!" should be shown
+        
+    Scenario: navigate matching for multiline links
+        when I open /data/navigate/multilinelinks.html
+        And I run ]]
```

Comment:
`And I run` runs commands, not keybindings. You can do `:bind ]]` to find out what something is bound to, so here that'd be `And I run :navigate next`.